src/audio/kpb.c

```diff
@@ -393,11 +393,11 @@ static int kpb_prepare(struct comp_dev *dev)
 	/* Initialize draining task */
 	schedule_task_init(&kpb->draining_task, /* task structure */
 			   SOF_SCHEDULE_EDF, /* utilize EDF scheduler */
-			   0, /* priority doesn't matter for IDLE tasks */
+			   SOF_TASK_PRI_IDLE - 1, /* almost idle priority */
```

Comment:
Lets have a mcro for this level, it protects users against internal scheduling changes.